domains/sunn4github.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "sunn4room",
+        "email": "sunn4room@163.com"
+    },
+    "record": {
+        "CNAME": "ghproxy.sunn4room.workers.dev"
```

Comment:
Little content provided.

Reply:
Completed !  [The Link](https://ghproxy.sunn4room.workers.dev)

Reply:
This just looks like an exact copy of GitHub homepage?

Reply:
Yes, it's a cf worker to access github. It's difficult to access github in China due to DNS attack. T.T ~~~

Reply:
Ah got it.